drivers/dfplayer/dfplayer_internal.c

```diff
@@ -75,9 +75,11 @@ static void _handle_playback_completed(dfplayer_t *dev, dfplayer_source_t src)
 
     dev->flags |= DFPLAYER_FLAG_NO_ACK_BUG;
 
-    if (dev->cb_done) {
+    uint32_t now_us = xtimer_now_usec();
+    if (dev->cb_done && (now_us - dev->last_event_us > US_PER_SEC)) {
```

Comment:
Better add a comment that this is to prevent duplicate events - otherwise the casual reader might wonder what this is for and even attempt to remove it.

Reply:
Good idea. Done